app/scripts/directives/oscFileInput.js

```diff
@@ -68,8 +69,12 @@ angular.module('openshiftConsole')
           });
         }
 
+        $('.clear-btn').on('click', function() {
```

Comment:
@jhadvig We should call a scope function from the existing ng-click handlerrather than using a jQuery event handler. Not good to mix these.